tests/train/test_tokenizer.py

```diff
@@ -0,0 +1,72 @@
+import collections
+import random
+
+from transformers import AutoTokenizer
+
+from delphi.eval.utils import load_validation_dataset
+from delphi.train.dataset_tokenization import (
+    extend_deque,
+    get_tokenized_batches,
+    make_new_samples,
+)
+
+tokenizer = AutoTokenizer.from_pretrained("delphi-suite/stories-tokenizer")
+
+
+def test_extend_deque():
+    CTX_SIZE = 10
+    dataset = load_validation_dataset("delphi-suite/tinystories-v2-clean")
```

Comment:
generate a random dataset

Reply:
Will review all tests once you have updated the test to not load validation dataset :)